core/src/main/java/org/elasticsearch/monitor/os/OsStats.java

```diff
@@ -374,7 +374,7 @@ public Cgroup(
 
         @Override
         public void writeTo(final StreamOutput out) throws IOException {
-            out.writeString(cpuAcctControlGroup);
+            out.writeOptionalString(cpuAcctControlGroup);
```

Comment:
That was a bug, I cleaned up all the optionality in 4c8a2cc72d7dddd460cc28117c6d1623f77374fb but missed this one. I pushed 1d53577c0254a37a26997459c4eb453d29011c18.